src/models/Following.model.ts

```diff
@@ -0,0 +1,40 @@
+import mongoose, { Schema, Document } from "mongoose";
+import paginate from "mongoose-paginate-v2";
+
+export interface Following extends Document {
+  followingId: mongoose.Schema.Types.ObjectId;
+  ownerId: mongoose.Schema.Types.ObjectId;
+  createdAt: string;
+  updatedAt?: string;
+}
+
+const FollowingSchema: Schema = new Schema<Following>({
+  followingId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+
+  ownerId: {
+    type: mongoose.Schema.Types.ObjectId,
```

Comment:
```suggestion
    type: Schema.Types.ObjectId,
```
https://mongoosejs.com/docs/typescript.html